maths/gelu.py

```diff
@@ -0,0 +1,49 @@
+"""
+This script demonstrates the implementation of the Gelu function.
+
+The function takes a vector x of K real numbers as input and
+resukts x * sigmoid(1.702*x).
+The Gaussian Error Linear Unit (GELU), is a high-performing neural
+network activation function.
+
+Script inspired from its corresponding research paper
+https://arxiv.org/abs/1606.08415
+"""
+
+import numpy as np
+
+
+def sigmoid(vector: np.array) -> np.array:
+    """
+    Gelu function can be implemented easily with the help of
+    sigmoid function
+    """
+    return 1 / (1 + np.exp(-vector))
+
+
+def gelu(vector: np.array) -> np.array:
```

Comment:
`Use self-documenting function names as discussed in CONTRIBUTING.md.
``suggestion
def gaussian_error_linear_unit(vector: np.array) -> np.array:
```